test/e2e/tests/tokens/increase-token-allowance.spec.js

```diff
@@ -272,10 +272,14 @@ describe('Increase Token Allowance', function () {
       tag: 'button',
       text: 'Next',
     });
-    driver.waitForSelector({
+    await driver.waitForSelector({
```

Comment:
this await was missing, but it's not the cause of the flakiness

Reply:
Nice catch.